### PR TITLE
Avoid protobuf 3.0.0.b2.post1.

### PR DESCRIPTION
**Pull request by tseaver**
See: https://github.com/GoogleCloudPlatform/gcloud-python/issues/1570

### CODE REVIEW

**Comment by dhermes** [comment]
Did you mean to close? I would say I couldn't verify the syntax for the `!=` but I assume you ran it.

**Comment by tseaver** [comment]
The syntax was right, but I ran into other stupidity, and lost the will to live. :)

**Comment by dhermes** [comment]
Branch has conflicts?

**Comment by dhermes** [comment]
Also the CLA check is the only check that ran

**Comment by tseaver** [comment]
https://github.com/grpc/grpc/issues/5280 defeated me.  I need to back out the forced use of grpcio in the `testenv:py27` section to move forward.

**Comment by tseaver** [comment]
@dhermes PTAL

File: tox.ini [hunk 1]
```diff
@@ -21,14 +21,17 @@ covercmd =
...
-basepython =
-    python2.7
-deps =
-    {[testenv]deps}
-    grpcio >= 0.13.0
-setenv =
-    PYTHONPATH =
+# Until grpcio 0.13.1 ships, this environment is broken on UCS2 builds.
+# See: https://github.com/grpc/grpc/issues/5280 and
+#      https://github.com/grpc/grpc/pull/5319
```
Comment (dhermes) [inline tox.ini]: Ditch the commented out stuff and just file an issue.

FYI @jgeewax

**Comment by dhermes** [comment]
LGTM. One day gRPC will be able to play with the big kids :)

File: tox.ini (hunk 1 above)
Comment (tseaver) [inline tox.ini]: I would normally agree, except that they [intend to release a fix early next week](https://github.com/grpc/grpc/pull/5319#issuecomment-192412023).

**Comment by bgenchel** [comment]
what is the actual fix here? brought here by #1570, saying the issue was fixed in this issue, but this conversation seems pretty unintelligible to me.

**Comment by dhermes** [comment]
Our fix is just to [avoid][1] a b0rken version that the `protobuf` maintainers uploaded to PyPI.

If you have a stacktrace that seems to contradict, please post it on that issue or open a new one.

[1]: https://github.com/GoogleCloudPlatform/gcloud-python/blob/5a6ba14fda655761d7104f3c8ef14d88d881de24/setup.py#L18

**Pull request merged**